libs/datasets/sources/nytimes_dataset.py

```diff
@@ -1,9 +1,12 @@
 from covidactnow.datapublic.common_fields import CommonFields
 from libs.datasets import data_source
+from libs.datasets.taglib import UrlStr
 
 
 class NYTimesDataset(data_source.DataSource):
     SOURCE_TYPE = "NYTimes"
+    SOURCE_NAME = "New York Times"
```

Comment:
I think this should be "The New York Times” https://github.com/nytimes/covid-19-data#license-and-attribution

Reply:
Fixed in feb9b826